poetry/console/commands/init.py

```diff
@@ -202,6 +220,10 @@ def handle(self):
         with (Path.cwd() / "pyproject.toml").open("w", encoding="utf-8") as f:
             f.write(content)
 
+            if original_toml:
```

Comment:
Might be a good idea to update `Layout.generate_poetry_content()` to accept base content and pass `original_toml` into it. Also, we should consider appending to the file.

Reply:
Yes updating `Layout.generate_poetry_content()` sounds like a good idea.

I also thought about appending to the file but decided to prepend because in my opinion the poetry stuff contains the metadata stuff for the project. And this is something I personally would expect at the beginning of the file.

Reply:
The issue with that is that we might end up pushing file comments down. In certain cases projects could have copyright/licence headers. imho, it is safer to append. This is somethign we can also tackle later.

Reply:
@finswimmer let me know your thoughts on this and I can go ahead approve this one.

Reply:
The point about the comments is a very good one. I guess I will change it later this week.